03 - Showcases & Templates/Vaults/🗂️ Vaults.md

```diff
@@ -9,6 +9,8 @@ tags:
 
 #placeholder/description
 
+[Blue topaz themes example vault for Obsidian](https://github.com/cumany/Blue-topaz-examples)
```

Comment:
Thanks!

**Thinking about location**

Is this linked vault showing how to structure a vault, or is it showcasing a theme?

I ask because this folder seems to be about sharing vault structure, rather than showcasing themes. 

If it is to showcase the theme, I wonder whether it might be worth adding a new folder "Theme Showcases"?

**Thinking about format**

If the new info does stay in this folder...

Reading the "Contributing" notes on this page, it seems the intention was to have a page-per-vault.

There is a template called "Vault Showcase" - and the existing notes match that format.

I'd suggest making a minimal page for this vault, and also in that page, adding a link to the theme's page, as the backlink will be useful.